yb-voyager/src/queryparser/traversal_plpgsql.go

```diff
@@ -120,47 +180,46 @@ parsed json -
 	  }
 	}
 */
-func TraversePlPgSQLActions(action interface{}, plPgSqlStatements *[]string) {
-	actionMap, ok := action.(map[string]interface{})
-	if !ok {
-		//In case the value of a field is not a <key , val> but a list of <key, val> e.g. "body"
-		lists, ok := action.([]interface{})
-		if ok {
-			for _, l := range lists {
-				TraversePlPgSQLActions(l, plPgSqlStatements)
-			}
-		}
-		return
-	}
-
-	for k, v := range actionMap {
-		switch k {
-		// base case of recursive calls to reach this PLPGSQL_EXPR field in json which will have "query" field with statement
-		case PLPGSQL_EXPR: 
-			expr, ok := v.(map[string]interface{})
-			if ok {
-				query, ok := expr[QUERY]
+func TraversePlPgSQLActions(fieldValue interface{}, plPgSqlStatements *[]string) {
```

Comment:
`TraversePlPgSQLJson` ?

Reply:
`TraversePlPgSQLJsonField`?

Reply:
both are okay. just because `fieldValue` is already an argument, imo `TraversePlPgSQLJson` suffices